### DISCUSSION TITLE
no support for svg

### DISCUSSION THREAD

**Discussion by Chranium** [yandeu/five-server#6] [Help / Questions] [answered]
does not show imported svg files with img tag

**Comment by yandeu** [comment, marked as answer]
Thanks for reporting!

I've found the issue. Expect this to be fixed in the next version!

**Reply by yandeu** [reply]
I have just published the latest version.

**Reply by Chranium** [reply]
Great, I just tested and it's working fine, thanks!

**Reply by yandeu** [reply]
You're welcome ☺️
